### PR TITLE
Updated Templar Armour

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
@duncathan merged the PR like 30 seconds before I could add Dorisdwarfs updated Templar sprites

Please merge this its a critical bug that templars dont have tabbards

Oh also I fixed a dumb line with deathsquad armour because it was in the file

### CODE REVIEW

**Comment by RemieRichards** [comment]
Gonna wait for Travis :P

**Comment by Dorsisdwarf** [comment]
Compile, travis, compile like the wind

**Comment by duncathan** [comment]
YOU SAID IT WAS MERGE-READY YOU SHIT

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -271,10 +271,11 @@ Contains:
 
 /obj/item/clothing/head/helmet/space/hardsuit/ert/paranormal
 	name = "paranormal response unit helmet"
-	desc = "A beaked hardsuit helmet, worn by those who deal with paranormal threats for a living."
+	desc = "A helmet worn by those who deal with paranormal threats for a living."
```
Comment (Iamgoofball) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: ghostbusters didnt wear helmets nerd

Comment (RemieRichards) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: these are 40K Grey Knights, not ghostbusters

**Comment by Dorsisdwarf** [comment]
He said to remove DNM honk

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: fuq

Comment (Dorsisdwarf) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: Demonhunters demonhunters demonhunters

Comment (KorPhaeron) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: *daemon

**Pull request merged**